app/main.py

```diff
@@ -1,2 +1,50 @@
+import dataclasses
+from typing import Any, Hashable
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.size: int = 0
+        self.capacity: int = 8
+        self.node = [None] * self.capacity
```

Comment:
make node private in this case because it possible that someone set value
and then if self.node will be full your code stop worker
Or inside __setitem__ do check for size before setting